src/openms/source/CHEMISTRY/ResidueDB.cpp

```diff
@@ -133,43 +132,42 @@ namespace OpenMS
     return s;
   }
 
-  void ResidueDB::setResidues_(const String& file_name)
+  void ResidueDB::initResidues_()
   {
     #pragma omp critical (ResidueDB)
     {
-      readResiduesFromFile_(file_name);
+      buildResidues_();
       buildResidueNames_();
     }     
   }
 
   void ResidueDB::addResidue_(Residue* r)
   {
     vector<String> names;
-    if (r->getName() != "")
+
```

Comment:
there are two different methods `addResidue_` and `insertResidue`, where is the difference?
Can you get rid of one?